lib/player.ml

```diff
@@ -0,0 +1,17 @@
+(* Type for each player in the list *)
+type player = { letters : char list }
+
+(** [nextplayer cur_player player_lst] gets the player after the player
```

Comment:
this design could be simplified by making it so that the list functions like a circular queue: the current player is always the player at the head of the list, and transitioning to the next player simply means removing the head and appending it at the tail. this way, you would not need a separate `cur_player` field in the state.